contrib/packages/deb/ubuntu-noble/debian/copyright

```diff
@@ -0,0 +1,116 @@
+This package was packaged for Debian by Brian P. Hinz <bphinz@users.sourceforge.net>
+on Tue, 18 Jun 2024 18:54:03 -0400 using the tightvnc package as a base.
```

Comment:
I hadn't noticed this file before. We're obviously not keeping this copy in sync properly. Can we automate that? Or better yet, just put a notice here to check the README?

Reply:
I don't know if there is an automated way to keep it in sync but I will take a look at what debian's packaging guidelines are regarding this file.  For now I've updated them all manually.